deployment/amplify/staging/amplify/.config/aws-info.json

```diff
@@ -0,0 +1,5 @@
+{
+    "configLevel": "project",
+    "useProfile": true,
+    "profileName": "echolocator"
```

Comment:
The Amplify CLI uses this attribute to source the AWS profile that you want to use. Not great, since you'll have to edit this file manually if your profile is named something different (in this case, the staging profile should correspond to the Azavea root AWS account, for instance).

Reply:
The deployment README says to configure a profile named 'echo-locator', with a hyphen.

Reply:
> Does this mean it will be necessary to edit the host AWS credentials to switch accounts/environments?

I'm imagining that if your profile name doesn't match what's in the `aws-info.json` config for the environment you're trying to update, you'll either need to rename your profile or edit the local copy of this config to match your profile name. I assume the latter (editing this config file) will be less obtrusive.

> The deployment README says to configure a profile named 'echo-locator', with a hyphen.

Seems like there's some inconsistency here, since the main repo uses `echolocator` (see https://github.com/azavea/echo-locator/blob/1c3b7da560546e08cb1f05cfc570587c42a04223/README.md#development). Another inconsistency I just remembered is that the staging stack is actually deployed to the Azavea root account. So I think our two options here are either:

1. Redeploy the staging stack in the BHA account and update all `aws-info.js` profile names to `echo-locator` (would remove any existing staging users/data, would be slower, but would be the cleanest setup)
2. Keep the staging stack in the Azavea root account and update this value to be e.g. `azavea`

Leaning towards 2) above, since it's the fastest move.